.github/workflows/workflow-common.yml

```diff
@@ -98,14 +98,6 @@ jobs:
           pip install --upgrade pip
           pip install -r ci/requirements.txt
 
-      - name: Build docker image for tests purpose
-        # Clean docker before test (github like to go out of memory)
-        # Also there is need for repeatable results with docker (cache is faster but it can give some false positives and false negatives while testing)
-        run: |
-          docker system prune -af --volumes
```

Comment:
Removin this is risky in my opinion, are you sure about that?

Reply:
Ok, now seems to be ok.

Reply:
Actually this was changed in quite current PR: https://github.com/Spyro-Soft/scargo/pull/292
Because we had problems with memory on CI